Makefile

```diff
@@ -51,6 +55,7 @@ all: default
 
 DEFINES   += OS_IO_SEPROXYHAL
 DEFINES   += HAVE_BAGL HAVE_SPRINTF
+DEFINES   += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=4 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
```

Comment:
Why are you enabling USB? I don't see why USB should be supported for plugins.

Reply:
I should not.